pkg/chartvalues/apiextensions_app_e2e_test.go

```diff
@@ -151,6 +152,7 @@ secrets:
 				return
 			}
 
+			fmt.Println(values)
```

Comment:
Is this needed?

Reply:
Maybe not a must have, guessing it's nice to have.